docs/semgrep-supply-chain/getting-started.md

```diff
@@ -45,64 +38,66 @@ In the following example, Semgrep Supply Chain assumes that all code files using
 ├───/biking
 ```
 
-If you have code files in `my-project/biking`, Semgrep Supply Chain does not associate them to the dependencies in `my-project/running/lockfile.json`. If there is another lockfile in `my-project/running`, such as `my-project/running/uphill/lockfile.json`, then this overrides the original `my-project/running/lockfile.json` for all code files in `my-project/running/uphill/` or deeper directories.
+If you have code files in `my-project/biking,` Semgrep Supply Chain does not associate them with the dependencies in `my-project/running/lockfile.json.` If there is another lockfile in `my-project/running`, such as `my-project/running/uphill/lockfile.json`, then this overrides the original `my-project/running/lockfile.json` for all code files in `my-project/running/uphill/` or deeper directories.
+
+:::info Apache Maven
+To run a Semgrep Supply Chain scan, generate a [dependency tree for Apache Maven](/semgrep-supply-chain/setup-maven).
+:::
 
 ## Enable Semgrep Supply Chain
 
 1. Sign in to [<i class="fas fa-external-link fa-xs"></i> Semgrep AppSec Platform](https://semgrep.dev/login).
 1. Click **[Settings](https://semgrep.dev/orgs/-/settings)**.
-1. In the **Deployment** tab, click the **<i class="fa-solid fa-toggle-large-on"></i> Supply Chain scans** toggle if it is not already enabled.
+1. In the **Deployment** tab, navigate to **Supply Chain (SCA)**, and click the **<i class="fa-solid fa-toggle-large-on"></i> Supply Chain scans** toggle if it is not already enabled.
 
 ## Scan frequency
 
-By adjusting your CI configuration, you can configure your scans so that Semgrep Supply Chain scans your code at a different frequency or when a specific event occurs.
-
-### Schedule scans
+You can modify your CI configuration so that Semgrep Supply Chain scans your code at a specified frequency or whenever a specific event occurs, such as opening a pull request or merge request.
```

Comment:
I suggest to add something like "If you have deployed Semgrep through CI..." because scan frequency is handled differently in Semgrep Managed Scans.